ImageFeed/Profile/ProfileViewController.swift

```diff
@@ -114,6 +118,42 @@ final class ProfileViewController: UIViewController {
     
     @objc
     func logoutButtonTap() {
-        print("New tep")
+        showAlert()
+    }
+    
+    private func showAlert() {
+        let alertController = UIAlertController(title: "Пока!",
+                                                message: "Уверены, что хотите выйти?",
+                                                preferredStyle: .alert)
+        alertController.addAction(UIAlertAction(title: "Да", style: .default, handler: { [weak self] action in
+            guard let self = self else { return }
+            self.logout()
+        }))
+        alertController.addAction(UIAlertAction(title: "Нет", style: .default, handler: nil))
+        present(alertController, animated: true, completion: nil)
+    }
+    
+    
+    private func logout() {
+        OAuth2TokenStorage().token = nil
```

Comment:
Эту строчку тоже можно вынести в ProfileService
Создать метод 
```
func cleanSession() {
    OAuth2TokenStorage().token = nil

    HTTPCookieStorage.shared.removeCookies(since: Date.distantPast)
         WKWebsiteDataStore.default().fetchDataRecords(ofTypes: WKWebsiteDataStore.allWebsiteDataTypes()) { records in
             records.forEach { record in
                 WKWebsiteDataStore.default().removeData(ofTypes: record.dataTypes, for: [record], completionHandler: {})
             }
         }
}
```